UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService/Helpers/ICallbackClient.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace UKHO.ExternalNotificationService.SubscriptionService.Helpers
+{
+    public interface ICallbackClient
+    {        
+        Task<HttpResponseMessage> GetCallbackD365Client(HttpMethod method, string externalEntityPath, string accessToken, object externalNotificationEntity, string correlationId, CancellationToken cancellationToken);
+    }
```

Comment:
Is it necessary to pass HttpMethod as parameter?

Reply:
Removed as a parameter